src/algorithms/reco/JetReconstruction.cc

```diff
@@ -105,13 +108,13 @@ namespace eicrecon {
     // Skip empty
     if (particles.empty()) {
       m_log->trace("  Empty particle list.");
-      return jet_collection;
+      return;
     }
     m_log->trace("  Number of particles: {}", particles.size());
 
     // Run the clustering, extract the jets
-    m_clus_seq = std::make_unique<ClusterSequenceArea>(particles, *m_jet_def, *m_area_def);
-    std::vector<PseudoJet> jets = sorted_by_pt(m_clus_seq->inclusive_jets(m_cfg.minJetPt));
+    fastjet::ClusterSequenceArea m_clus_seq(particles, *m_jet_def, *m_area_def);
+    std::vector<PseudoJet> jets = sorted_by_pt(m_clus_seq.inclusive_jets(m_cfg.minJetPt));
```

Comment:
I'm still mystified that the pointer version doesn't compile with the changes... 🤔 

Either way, if we do implement this change, please don't forget to remove the corresponding `std::unique_ptr<etc> m_clus_seq` from the header!